apps/dashboard/app/views/projects/edit.html.erb

```diff
@@ -1,7 +1,18 @@
-<div class='page-header text-center'>
-  <h1>Edit Project</h1>
+<div class='page-header text-center pb-3'>
+  <h1>
+    Editing: <%= @project.title %>
+  </h1>
 </div>
 
-<br>
+<%= bootstrap_form_for(@project, url: project_path, method: :patch) do |form| %>
+  <%= render "project_edit_form", form: form %>
```

Comment:
I can't figure out how to combine these. I had used the 2 forms to make `dir` available in the `new` but not available in the `edit` since the `project_edit_form.html.erb` has no fields for `dir` which was the pattern i thought `products` view for `edit` was using looking at `form_git` and `form_manifest` (which I what we renamed to `form`).